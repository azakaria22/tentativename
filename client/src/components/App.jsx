import React, { Component } from 'react';
import PropTypes from 'prop-types';
import Expenses from './Expenses.jsx';

class App extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
        <h1>Slytherin</h1>
        <div>
          Bills
          Savings
          Loans
          Investments
          Settings
        </div>
        <Expenses />
      </div>
    )
  }
}

export default App;