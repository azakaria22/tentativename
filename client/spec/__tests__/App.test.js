import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../../src/components/App.jsx';

describe('<App />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const component = shallow(<App />);
      expect(toJson(component)).toMatchSnapshot();
    })
  })
})