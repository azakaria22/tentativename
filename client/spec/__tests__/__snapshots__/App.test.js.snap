// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> render() renders the component 1`] = `
<div>
  <h1>
    Moneyway
  </h1>
  <div>
    Bills Savings Loans Investments Settings
  </div>
  <Expenses />
</div>
`;
